### PR TITLE
Throw exception when not use swsssdk in UT.

### PR DESCRIPTION
**Pull request by liuh-80**
#### Why I did it
swsssdk been deprecated after 202205 branch, all use should switch to sonic-swss-common.

#### How I did it
Throw exception when load swsssdk for none UT usage.

#### How to verify it
Pass all UT.

#### Description for the changelog
Throw exception when load swsssdk for none UT usage.

### CODE REVIEW

**Comment by liuh-80** [comment]
E2E test will validate with this PR: https://github.com/sonic-net/sonic-buildimage/pull/12036

**Comment by liuh-80** [comment]
Still some UT blocked by this change:

2022-09-09T10:45:45.6386792Z test_backend_acl_template_render (tests.test_j2files.TestJ2Files) ... Traceback (most recent call last):
2022-09-09T10:45:45.6387777Z   File "/sonic/src/sonic-config-engine/tests/../sonic-cfggen", line 32, in <module>
2022-09-09T10:45:45.6388280Z     from minigraph import minigraph_encoder, parse_xml, parse_device_desc_xml, parse_asic_sub_role, parse_asic_switch_type
2022-09-09T10:45:45.6388953Z   File "/sonic/src/sonic-config-engine/minigraph.py", line 15, in <module>
2022-09-09T10:45:45.6389302Z     from portconfig import get_port_config
2022-09-09T10:45:45.6389800Z   File "/sonic/src/sonic-config-engine/portconfig.py", line 20, in <module>
2022-09-09T10:45:45.6390623Z     import mock_tables.dbconnector
2022-09-09T10:45:45.6391865Z   File "/sonic/src/sonic-config-engine/./tests/mock_tables/dbconnector.py", line 10, in <module>
2022-09-09T10:45:45.6392526Z     import swsssdk
2022-09-09T10:45:45.6393590Z   File "/usr/local/lib/python2.7/dist-packages/swsssdk/__init__.py", line 23, in <module>
2022-09-09T10:45:45.6394254Z     raise RuntimeError(msg)
2022-09-09T10:45:45.6395454Z RuntimeError: sonic-py-swsssdk been deprecated, please switch to sonic-swss-common.

**Comment by liuh-80** [comment]
This PR depends on another PR merge first: https://github.com/sonic-net/sonic-buildimage/pull/12185

**Comment by liuh-80** [comment]
The code change in this PR validated with this PR: https://github.com/sonic-net/sonic-buildimage/pull/12036

File: test/test_moduleLoad.py [hunk 1]
```diff
@@ -34,3 +34,14 @@ def test__dbConfig(self):
...
             self.assertEqual(dbConfig.get_dbid('APPL_DB', namespace), 0)
+
+def test_BlockUseSwsssdk():
+    # change working folder to import swsssdk in subprocess
+    swsssdk_path = os.path.join(modules_path, 'src')
+    wd = os.getcwd()
+    os.chdir(swsssdk_path)
+
+    result = subprocess.run(["python", "-c", "import swsssdk;exit()"], capture_output=True)
+    assert "deprecated" in result.stderr.decode("utf-8")
```
Comment (qiluo-msft) [inline test/test_moduleLoad.py:45]: >assert [](http://example.com/codeflow?start=4&length=6)

if assert False, the next statement will be skipped, and there is side effect.

File: src/swsssdk/__init__.py [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
 """
+import sys
 import logging
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 logger.addHandler(logging.NullHandler())
 
+if ('unittest' not in sys.modules.keys() and
+    'mockredis' not in sys.modules.keys() and
```
Comment (qiluo-msft) [inline src/swsssdk/__init__.py:12]: >'mo [](http://example.com/codeflow?start=4&length=3)

Check coding style: [https://stackoverflow.com/a/5253378/2514803](https://stackoverflow.com/a/5253378/2514803)

File: setup.py [hunk 3]
```diff
@@ -26,12 +26,6 @@
     extras_require={
         'high_perf': high_performance_deps
     },
-    entry_points={
-        'console_scripts': [
```
[the remark this answers is not in the document]
Comment (liuh-80) [reply setup.py:30]: Fixed

File: src/swsssdk/__init__.py (hunk 2 above)
Comment (liuh-80) [reply src/swsssdk/__init__.py]: Fixed

File: test/test_moduleLoad.py (hunk 1 above)
Comment (liuh-80) [reply test/test_moduleLoad.py]: Fixed

File: setup.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (qiluo-msft) [reply setup.py:30]: Sorry for misleading. `console_scripts` variable should be defined at file global scope. It is not a parameter for `setup` function.

File: test/test_moduleLoad.py [hunk 4]
```diff
@@ -34,3 +34,11 @@ def test__dbConfig(self):
         for namespace in list(dbConfig.get_ns_list()):
             self.assertEqual(dbConfig.get_dbid('PFC_WD_DB', namespace), 5)
             self.assertEqual(dbConfig.get_dbid('APPL_DB', namespace), 0)
+
+def test_BlockUseSwsssdk():
+    # change
```
Comment (qiluo-msft) [inline test/test_moduleLoad.py:39]: >change [](http://example.com/codeflow?start=6&length=6)

You want to remove the whole comment line?

File: setup.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (liuh-80) [reply setup.py:30]: Fixed.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**